lambdas/post_to_slack/post_to_slack.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+
+"""
+
+import json
+import pprint
+import os
```

Comment:
I wonder if we can write a bot to make these comments?
But isn't that what linting is for?